include/boost/type_traits/type_with_alignment.hpp

```diff
@@ -120,7 +120,8 @@ template<> struct is_pod< ::boost::tt_align_ns::a128> : public true_type{};
 // registers.  Therefore we extend type_with_alignment<> to support
 // such types, however, we have to be careful to use a builtin type
 // whenever possible otherwise we break previously working code:
-// see http://article.gmane.org/gmane.comp.lib.boost.devel/173011
+// see https://groups.google.com/g/boost-developers-archive/c/cM9xfiqkwLo
```

Comment:
The better link is: https://lists.boost.org/Archives/boost/2014/03/212391.php (No need to retain the "originally" part).